opam.export

```diff
@@ -1,5 +1,4 @@
 opam-version: "2.0"
```

Comment:
why is this file changed relative to dev?

Reply:
Probably for logs or tezt. Any relevant changes should've been inherited from one of those two branches directly and can be undone safely.

Reply:
fixed by 9a8f1049c